.github/workflows/promote_to_module-manifests.yaml

```diff
@@ -1,27 +1,33 @@
 name: Promote to channel via module-manifests
 
 env:
-  # repositories urls
-  IMAGE_REPO: europe-docker.pkg.dev/kyma-project/prod/btp-manager
-  MODULES_REPO: "https://github.tools.sap/kyma/modules-manifests"
-  MODULE_MANIFESTS_WITH_USER: "https://kyma-btp-manager-serviceuser:${{ secrets.BOT_TOKEN_GITHUB_TOOLS }}@github.tools.sap/kyma/module-manifests.git"
-  DOC_URL: "https://kyma-project.io/#/btp-manager/user/README"
-  BTP_MANAGER_REPO_URL: "https://api.github.com/repos/kyma-project/btp-manager"
+  # git configuration
+  GIT_USER_EMAIL: <>
+  GIT_USER_NAME: kyma-btp-manager-serviceuser
```

Comment:
we can use GA variables: https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows